embedchain/config/apps/AppConfig.py

```diff
@@ -32,6 +33,7 @@ def __init__(self, log_level=None, host=None, port=None, id=None, collection_nam
             port=port,
             id=id,
             collection_name=collection_name,
+            anonymous_telemetry=anonymous_telemetry,
```

Comment:
lets call this flag as `collect_metrics`

reasoning: "Metrics" is a broad term that can encompass various kinds of quantitative data. "Collect" explicitly describes the action of gathering data, making the purpose of the flag straightforward. we want to use this metrics to improve our package and everything will be anonymized.